# 055.py - 예외처리 이해하기(try~except)[1]

try:
    print('안녕하세요')
    print(param)
except:
    print('예외가 발생했습니다!')